### PR TITLE
Update deployment; new image has call-home script and uses newer code.

### PR DESCRIPTION
**Pull request by ryanreich**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ryanreich__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@ryanreich Looks like there are merge conflicts as well. Can you take a look at those as well? Thanks!

**Comment by ryanreich** [comment]
@singhkay I don't get the problem with the parameter names. Aren't I allowed to add or change them?  Also not sure why metadata.json is problematic, because I didn't touch that string.

**Comment by ryanreich** [comment]
Anyway, I made changes that should have fixed those.

File: blockapps-strato/azuredeploy.json [hunk 1]
```diff
@@ -8,55 +8,60 @@
                 "description" : "What to call this virtual machine"
             }
         },
-        "storageAccount": {
-            "type": "string",
-            "metadata": {
-                "description" : "The storage account in which to create this VM"
+	"contactEmail": {
```
Comment (singhkay) [inline blockapps-strato/azuredeploy.json]: Formatting is off. Please pass these files through a JSON formatter

File: blockapps-strato/azuredeploy.json [hunk 2]
```diff
@@ -8,55 +8,60 @@
...
+	    }
+	},
+        "password" : {
+            "type" : "secureString",
+            "metadata" : {
+                "description" : "Password for admin account 'strato'"
             }
         },
-        "blockchain": {
+        "storageAccount": {
```
Comment (singhkay) [inline blockapps-strato/azuredeploy.json]: Please eliminate storage account parameter. See #944 for guidance

**Comment by ryanreich** [comment]
I am leaving the storage account parameter because the solution at #944 can also produce invalid names.  Perhaps azure needs a template function for this (or, better, to abandon the very odd restrictions on storage account names).

I fixed the formatting issues: tabs, for some reason.

**Comment by singhkay** [comment]
@ryanreich Looks like your ran into the profanename filter. Good news is there's been work to remove a bunch of stuff off that list, so the unique string generated names should pass. The fix will be deployed in a couple of weeks so it's better to use the #944 guidance

**Comment by ryanreich** [comment]
If it's not going to work for a couple of weeks then I will make the change
then; this template has to work, though.  If that's the only change you
would want me to make, please merge now and I will update in March.
ᐧ

**Comment by ryanreich** [comment]
I need to emphasize that this template has to be functional, and currently,
it is not.  These changes have to be merged.
ᐧ

File: blockapps-strato/azuredeploy.parameters.json [hunk 3]
```diff
@@ -2,20 +2,20 @@
     "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "instanceName": {
-            "value": "strato-deployment"
-        },
-        "blockchain": {
-            "value": "live"
+        "deploymentName": {
```
Comment (singhkay) [inline blockapps-strato/azuredeploy.parameters.json]: See CI guidance https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci

File: blockapps-strato/azuredeploy.json [hunk 4]
```diff
@@ -2,61 +2,66 @@
...
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "instanceName": {
+        "deploymentName": {
             "type": "string",
             "metadata": {
                 "description" : "What to call this virtual machine"
             }
         },
-        "storageAccount": {
```
Comment (singhkay) [inline blockapps-strato/azuredeploy.json]: Call this newStorageAccount

**Comment by ryanreich** [comment]
Honestly not sure what's wrong with the CI; doing `azure group template validate` returns OK, and the CI build log is uninformative.

**Comment by singhkay** [comment]
@ryanreich Looks like a minor blip in CI server availability. It said server not found but I ran the deployment again and it passed. Btw `azure group template validate` only validates for syntactical correctness, doesn't actually deploy the template.

**Pull request merged**